.config/stylelintrc.json

```diff
@@ -5,10 +5,11 @@
   ],
 
   "rules": {
+    "color-function-notation": "legacy",
     "selector-class-pattern": null,
-    "scss/no-global-function-names": null,
+    "shorthand-property-no-redundant-values": null,
```

Comment:
I actually like the long form that we use in a few places, so I opted to disable this.